src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "Alby - Bitcoin Lightning Wallet",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
fyi: the version is set by the package.json

Reply:
Ah, thanks